tests/Oryx.Integration.Tests/Php/PhpSqlServerIntegrationTests.cs

```diff
@@ -29,6 +29,16 @@ public PhpSqlServerIntegrationTests(ITestOutputHelper output) : base(output, nul
         // Unique category traits are needed to run each
         // platform-version in it's own pipeline agent. This is
         // because our agents currently a space limit of 10GB.
+        [Fact, Trait("category", "php-8.1")]
+        public async Task PipelineTestInvocationsPhp81Async()
```

Comment:
It looks like this test is failing in the nightly build -- would you mind taking a look?